admin-cli/cmd/duplication.go

```diff
@@ -49,11 +49,11 @@ func init() {
 			if c.Flags.String("cluster") == "" {
 				return fmt.Errorf("cluster cannot be empty")
 			}
-			return executor.AddDuplication(pegasusClient, c.UseTable, c.Flags.String("cluster"), c.Flags.Bool("freezed"))
+			return executor.AddDuplication(pegasusClient, c.UseTable, c.Flags.String("cluster"), c.Flags.Bool("sst"))
```

Comment:
why not naming `duplicateCheckpoint`?

Reply:
duplication v2 has removed `freezed`, and add `whether duplicate checkpoint`